queue/src/fiskaltrust.Middleware.Localization.QueueDE/SignProcessorDE.cs

```diff
@@ -38,7 +38,7 @@ public class SignProcessorDE : IMarketSpecificSignProcessor
         public async Task<(ReceiptResponse receiptResponse, List<ftActionJournal> actionJournals)> ProcessAsync(ReceiptRequest request, ftQueue queue, ftQueueItem queueItem)
         {
             _logger.LogTrace("SignProcessorDE.ProcessAsync called.");
-            if (string.IsNullOrEmpty(request.cbReceiptReference) && !request.IsFailTransactionReceipt() && !string.IsNullOrEmpty(request.ftReceiptCaseData) && !request.ftReceiptCaseData.Contains("CurrentStartedTransactionNumbers"))
+            if(request.IsFailTransactionReceipt() && (string.IsNullOrEmpty(request.cbReceiptReference) || !request.ftReceiptCaseData.Contains("CurrentStartedTransactionNumbers")))
```

Comment:
@forsthug @volllly does someone remember why this check is here, and not in the `FailTransactionReceiptCommand`?